pacemaker_metrics.go

```diff
@@ -237,6 +243,16 @@ func (c *pacemakerCollector) recordFailCountMetrics(pacemakerStatus pacemakerSta
 	}
 }
 
+func (c *pacemakerCollector) recordResourceAgentsChanges(pacemakerStatus pacemakerStatus, ch chan<- prometheus.Metric) {
+	t, err := time.Parse(time.ANSIC, pacemakerStatus.Summary.LastChange.Time)
+	if err != nil {
+		log.Warnln(err)
+		return
+	}
+	// is the resource have changed we set a different timeout from pacemaker
+	ch <- c.makeGaugeMetric("resource_configuration_changes", float64(t.Unix()))
```

Comment:
I'm not super convinced on how we are modelling this metric, to be frank.
IMHO, representing the time in a float, when we already have an integer timestamp at the end of the line, is super-confusing for the user. One possible solution would be to always set the value to `0`, and use the timestamp itself.

eg:
```suggestion
	ch <- prometheus.NewMetricWithTimestamp(t, prometheus.MustNewConstMetric(c.metrics["resource_configuration_changes"], prometheus.CountValue, 0))
```

What are your thoughts on this?

Reply:
I agree that is a tricky metrics :grin: . let mes seartch about the timestamp.

Our goal is to indicate a change so an user can detect that with promql functions.